pkg/e2e/compose_test.go

```diff
@@ -135,6 +135,9 @@ func TestDownComposefileInParentFolder(t *testing.T) {
 }
 
 func TestAttachRestart(t *testing.T) {
+	if _, ok := os.LookupEnv("CI"); ok {
+		t.Skip("Skipping test on CI... flaky")
```

Comment:
Same for this test, and same regarding always passing on the M1 mac but being flaky on the Intel mac/Windows runner.

Reply:
Oof yeah I've been 👀 this test for a while, might take a look tomorrow to see if I can figure out where the race is

Reply:
(To be clear though, I'm in favor of merging this with the `Skip()` and deal with cleanup after!)